src/main/scala/com/gu/invoicing/pdf/README.md

```diff
@@ -0,0 +1,65 @@
+## Get invoice PDF by invoiceId
+
+The endpoint is secured with 
+- API key
+- AWS Signature
+- Validation that invoice belongs to identity that requests it 
+
+**Request:**
+
+```
+GET /invoices/{invoiceId}
+Host: https://{apiGatewayId}.execute-api.{region}.amazonaws.com/{STAGE}
+x-identity-id
+x-api-key: ********
+X-Amz-Security-Token: ******** 
+X-Amz-Date: ********
+Authorization: ******** 
+```
+
+**Response:**
+
+Base64 encoded PDF:
+
+```
+Content-Type: application/pdf;charset=UTF-8
+```
+
+### How to test in CODE
+
+#### With CLI 
+
+1. Get invoicing-api+uat@guardian.co.uk Zuora OAuth client credentials
+1. Follow docs in `Cli.scala` and create `Stage` and `Config` environmental variables
+1. Simply run something like
+    ```
+    program(PdfInput("anInvoiceId", "anIdentityId"))
+    ```
+
+#### With Lambda
+
+1. Get fresh Janus credentials
+1. `Program.scala` contains main business logic 
+1. `deployAwsLambda` will upload modified lambda package
+1. In AWS Lambda console create test event that reflects `GET /invoices/{invoiceId}`
+    ```
+    {
+      "headers": {
+        "x-identity-id": "1000001"
+      },
+      "pathParameters": {
+        "invoiceId": "1a2s3d4f5g6h7j8k"
+      }
+    }
+    ```
+1. Tail logs 
+    ```
+    export AWS_DEFAULT_REGION=eu-west-1 && awslogs get --profile membership /aws/lambda/invoicing-api-pdf-CODE --watch
+    ```
+   
+#### With API Gateway (Postman)
+
+1. HTTP request will have to be signed with AWS signature
+1. Get fresh janus credentials
+1. Paste them in `Postman | Auth | AWS Signature`
```

Comment:
I don't understand this step

Reply:
@kelvin-chappell I mean this

![image](https://user-images.githubusercontent.com/13835317/90380915-9347b600-e074-11ea-9d85-9ecc02ddc52d.png)